src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Person } from '../../types';
+import { useParams } from 'react-router-dom';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { personSlug } = useParams();
+
+  const findPerson = (name: string) => {
+    const person = people.find(obj => obj.name === name);
+
+    return person ? <PersonLink person={person} /> : name;
```

Comment:
The `findPerson` function returns a `PersonLink` component if a person is found. Ensure that the `PersonLink` component is correctly handling the `person` prop and rendering as expected. If `PersonLink` is not defined to handle the `person` object, this could lead to unexpected behavior.